src/dappsExplorer/NoResults.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import Touchable from 'src/components/Touchable'
+import { DappFilter } from 'src/dapps/types'
+import { currentLanguageSelector } from 'src/i18n/selectors'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  filter: DappFilter | null
+  removeFilter: () => void
+  testID: string
+}
+
+export function NoResults({ filter, removeFilter, testID }: Props) {
+  const { t } = useTranslation()
+  const language = useSelector(currentLanguageSelector)
+
+  const onPress = () => {
+    removeFilter()
+  }
+
+  return (
+    <View testID={`${testID}/NoResults`} style={styles.favoritesContainer}>
+      <Text style={styles.filterAppliedText}>
+        <Trans
+          i18nKey={'dappsScreen.emptyResults.message'}
+          tOptions={{ filter: filter?.name.toLocaleUpperCase(language ?? 'en-US') }}
+        >
+          <Text style={styles.filterText} />
+        </Trans>
+      </Text>
+      <View style={styles.removeFilterTouchableContainer}>
+        <Touchable
+          hitSlop={{ top: 10, bottom: 10, left: 10, right: 10 }}
```

Comment:
nit: there is an `iconHitslop` in the variables file for this

Reply:
Fixed in 3347408038796b8fd990bacb10192130b069ddd2!